### PR TITLE
Allow customization of type labels

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

**Comment by emosbaugh** [comment]
[sc-58043]

File: type_label_flags.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	"github.com/google/go-github/v43/github"
+)
+
+type TypeLabelFlags struct {
+	FeatureTypeLabels, ImprovementTypeLabels, BugTypeLabels                   []string
+	featureTypeLabelsString, improvementTypeLabelsString, bugTypeLabelsString string
+}
+
+func NewTypeLabelFlags(flags *flag.FlagSet) *TypeLabelFlags {
```
[the remark this answers is not in the document]
Comment (sgalsaleh) [reply type_label_flags.go:15]: nevermind, i just saw your comment in the code below.

**Review by sgalsaleh — APPROVED** [review]

**Review by e3b0c442 — APPROVED** [review]
LGTM, would like to have Salah review the

File: type_label_flags.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (sgalsaleh) [reply type_label_flags.go:15]: also, looks like defaults would come from the action.yml file, so that should be sufficient

**Pull request merged**